### PR TITLE
feat: new payment invoice module

### PR DESCRIPTION
**Pull request by polymahh**
## Description

this is a new module for the payment invoice 

## Related Issues

https://github.com/idurar/erp-crm/issues/513

## Screenshots (if applicable)

- read page

image

- update page

image

create can only be done from invoice/pay

### CODE REVIEW

File: frontend/src/forms/PaymentInvoiceForm.jsx [hunk 1]
```diff
@@ -44,10 +44,10 @@ export default function PaymentInvoiceForm({ maxAmount = null, isUpdateForm = fa
         {/* <InputNumber
           className="moneyInput"
           min={0}
-          max={maxAmount}
           controls={false}
-          addonAfter={money.currencyPosition === 'after' ? money.currencySymbol : undefined}
-          addonBefore={money.currencyPosition === 'before' ? money.currencySymbol : undefined}
+          max={maxAmount}
+          addonBefore={money.currencyPosition === 'before' ? money.currencySymbol : null}
+          style={{ textAlign: 'left' }}
```
Comment (salahlalami) [inline frontend/src/forms/PaymentInvoiceForm.jsx:50]: keep this
`          addonAfter={money.currencyPosition === 'after' ? money.currencySymbol : undefined}`

**Review by salahlalami — CHANGES_REQUESTED** [review]

File: frontend/src/forms/PaymentInvoiceForm.jsx (hunk 1 above)
Comment (polymahh) [reply frontend/src/forms/PaymentInvoiceForm.jsx:50]: do you want me uncomment it ?

Comment (salahlalami) [reply frontend/src/forms/PaymentInvoiceForm.jsx:50]: yes

**Comment by polymahh** [comment]
@salahlalami like this ?

image

**Review by salahlalami — APPROVED** [review]

**Review by Ando22 — APPROVED** [review]
LGTM

**Pull request merged**